CLI/Operation/Define.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Rfd\ImageMagick\CLI\Operation;
+
+use Rfd\ImageMagick\Image\Image;
+
+class Define extends \Rfd\ImageMagick\Operation\Define {
+    /**
+     * @param Image $image
+     * @param string $command_line
+     *
+     * @return Result
+     */
+    public function process(Image $image, $command_line = '') {
+        
+        foreach($this->definition as $key=>$value)
```

Comment:
Code style:
Missing spaces between `foreach` and brackets.
Missing spaces around `=>`.
Open curly brace is on the next line.